lib/chef-dk/cli.rb

```diff
@@ -84,13 +86,29 @@ def run
     def handle_options
       parse_options(argv)
       if config[:version]
-        msg("Chef Development Kit Version: #{ChefDK::VERSION}")
+        show_version
       else
         show_help
       end
       exit 0
     end
 
+    def show_version
+      msg("Chef Development Kit Version: #{ChefDK::VERSION}")
+
+      ["chef-client", "knife", "berks", "kitchen"].each do |component|
+        result = nil
+        Bundler.with_clean_env do
+        result = shell_out("#{component} --version")
+      end
```

Comment:
Cool, I'm new to ruby so Claire showed me how to be a spacing wizard now. Updated in new commit.